contacts.js

```diff
@@ -4,46 +4,65 @@ const shortid = require("shortid");
 
 const contactsPath = path.resolve(__dirname, "db", "contacts.json");
 
-function listContacts() {
-  fs.readFile(contactsPath)
-    .then((data) => console.table(JSON.parse(data)))
+async function listContacts() {
+  return await fs
+    .readFile(contactsPath)
+    .then((data) => JSON.parse(data))
     .catch((err) => console.log("Can't get contacts list: ", err));
 }
 
-function getContactById(contactId) {
-  fs.readFile(contactsPath)
+async function getContactById(contactId) {
+  return await fs
+    .readFile(contactsPath)
     .then((data) =>
-      console.log(JSON.parse(data).find((contact) => contact.id === contactId))
+      JSON.parse(data).find((contact) => contact.id === contactId)
     )
     .catch((err) => console.log("Can't get a contact: ", err));
 }
 
-function removeContact(contactId) {
-  fs.readFile(contactsPath)
+async function removeContact(contactId) {
+  let filteredContacts = [];
+  await fs
+    .readFile(contactsPath)
     .then((data) => {
-      const filteredContacts = JSON.parse(data).filter(
+      filteredContacts = JSON.parse(data).filter(
         (contact) => contact.id !== contactId
       );
-      console.table(filteredContacts);
       fs.writeFile(contactsPath, JSON.stringify(filteredContacts));
     })
     .catch((err) => console.log("Can't delete a contact: ", err));
+
+  return filteredContacts;
 }
 
-function addContact(name, email, phone) {
-  fs.readFile(contactsPath)
+async function addContact({ name, email, phone }) {
+  const newContact = { id: shortid.generate(), name, email, phone };
+  await fs

```

Comment:
remove all then style promises

you should use only async/await style in all files